src/mongo/db/encryption/SConscript

```diff
@@ -91,10 +85,7 @@ kmipClientEnv = env.Clone()
 kmipClientEnv.InjectThirdParty(libraries=['libkmip', 'kmippp'])
 kmipClientEnv.Library(
     target='kmip_client',
-    source=[
-        'kmip_client.cpp',
-        'kmip_exchange.cpp'
-    ],
+    source=['kmip_client.cpp', 'kmip_exchange.cpp'],
```

Comment:
Again, it is probably no comma after `'kmip_exchange.cpp'` that leads to this formatting change.